pkg/volume/csi/csi_util.go

```diff
@@ -203,3 +204,17 @@ func createCSIOperationContext(volumeSpec *volume.Spec, timeout time.Duration) (
 	ctx := context.WithValue(context.Background(), additionalInfoKey, additionalInfo{Migrated: strconv.FormatBool(migrated)})
 	return context.WithTimeout(ctx, timeout)
 }
+
+// GetPodInfoAttrs returns pod info for NodePublish
+func GetPodInfoAttrs(pod *api.Pod, volumeMode storage.VolumeLifecycleMode) map[string]string {
```

Comment:
Does this need to be public? It's within the same package.

Reply:
@msau42 resloved～